app/src/main/java/com/yello/presentation/onboarding/fragment/school/SchoolAdpapter.kt

```diff
@@ -0,0 +1,43 @@
+package com.yello.presentation.onboarding.fragment.school
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.example.domain.entity.MySchool
+import com.example.ui.view.ItemDiffCallback
+import com.yello.databinding.ItemSchoolListBinding
+import timber.log.Timber
+
+class SchoolAdpapter : ListAdapter<MySchool, SchoolAdpapter.SchoolViewHolder>(diffUtil) {
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): SchoolViewHolder {
+        Timber.d("onCreateViewHolder")
+        return SchoolViewHolder(
+            ItemSchoolListBinding.inflate(
+                LayoutInflater.from(parent.context),
+                parent,
+                false,
+            ),
+        )
+    }
+
+    override fun onBindViewHolder(holder: SchoolViewHolder, position: Int) {
+        Timber.d("onBindViewHolder($position)")
+        holder.setlist(getItem(position))
+    }
+
+    class SchoolViewHolder(private val binding: ItemSchoolListBinding) :
+        RecyclerView.ViewHolder(binding.root) {
+        fun setlist(list: MySchool) {
+            Timber.d("set list : $list")
+            binding.data = list
```

Comment:
```suggestion
        fun setlist(school: MySchool) {
            binding.data = school
```
학교 하나에 대한 변수가 인자로 전달되는데 `list`라는 네이밍은 조금 부적합한 것 같습니다!